test/commands/user/reputation_token/award_for_pull_request_test.rb

```diff
@@ -10,7 +10,7 @@ class User::ReputationToken::AwardForPullRequestTest < ActiveSupport::TestCase
     url = 'https://api.github.com/repos/exercism/v3/pulls/1347'
     html_url = 'https://github.com/exercism/v3/pull/1347'
     labels = []
-    user = create :user, handle: "User22", github_username: "user22"
+    user = create :user, handle: "User-22", github_username: "user22"
```

Comment:
I've updated the test data to have users have a different handle from their github_username.